common/src/main/resources/db/changelog/v001/add_sbx_accounts.sql

```diff
@@ -0,0 +1,52 @@
+INSERT INTO sponsor (id, hpms_id, org_name, legal_name, parent_id) VALUES ((select nextval('hibernate_sequence')), 2, 'PDP-100', null, null);
+INSERT INTO sponsor (id, hpms_id, org_name, legal_name, parent_id) VALUES ((select nextval('hibernate_sequence')), 3, 'PDP-1000', null, null);
+INSERT INTO sponsor (id, hpms_id, org_name, legal_name, parent_id) VALUES ((select nextval('hibernate_sequence')), 4, 'PDP-2000', null, null);
+INSERT INTO sponsor (id, hpms_id, org_name, legal_name, parent_id) VALUES ((select nextval('hibernate_sequence')), 5, 'PDP-5000', null, null);
+INSERT INTO sponsor (id, hpms_id, org_name, legal_name, parent_id) VALUES ((select nextval('hibernate_sequence')), 6, 'PDP-10000', null, null);
+INSERT INTO sponsor (id, hpms_id, org_name, legal_name, parent_id) VALUES ((select nextval('hibernate_sequence')), 7, 'PDP-30000', null, null);
```

Comment:
These are all "parent" organizations (due to parent_id being null), and in our design parent orgs cannot have direct links to contracts. Let's create a single parent org called `PDP-SBX` and make all these children on that. See how Eileen F. was set up.